languages.yaml

```
@@ -138,6 +138,38 @@ Assembly:
       - yellow
       - green
     chip: "#6E4C13"
+Astro:
+  type: programming
+  ascii: |                                               
+    {0}+++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    {0}++++++++++++++++++++++*{1}@@@@@@@@@{0}#++++++++++++++++++++++
+    {0}++++++++++++++++++++++{1}@@@@@@@@@@@{0}#+++++++++++++++++++++
+    {0}+++++++++++++++++++++%{1}@@@@@@@@@@@@{0}+++++++++++++++++++++
+    {0}++++++++++++++++++++#{1}@@@@@@#@@@@@@{0}%++++++++++++++++++++
+    {0}++++++++++++++++++++{1}@@@@@@#+*{1}@@@@@@{0}*+++++++++++++++++++
+    {0}+++++++++++++++++++%{1}@@@@@@{0}++++{1}%@@@@@@{0}+++++++++++++++++++
+    {0}++++++++++++++++++*{1}@@@@@@{0}*+++{1}+@@@@@@{0}#++++++++++++++++++
+    {0}++++++++++++++++++{1}@@@@@@{0}%++++++{1}#@@@@@@{0}++++++++++++++++++
+    {0}+++++++++++++++++#{1}@@@@@@{0}*++++++{1}@@@@@@{0}%+++++++++++++++++
+    {0}+++++++++++++++++{1}@@@@@@{0}#+++++++*{1}@@@@@@{0}*++++++++++++++++
+    {0}++++++++++++++++%{1}@@@%%{0}#+++++++++{1}##%@@@@{0}++++++++++++++++
+    {0}++++++++++++++++{1}#*{0}+++++++++++++++++++{1}*#*{0}+++++++++++++++
+    {0}++++++++++++++++++++++{2}#*{0}++++++++{2}**{0}+++++++++++++++++++++
+    {0}++++++++++++++++++++++{2}%%%%%%%%%%%*{0}+++++++++++++++++++++
+    {0}++++++++++++++++++++++{2}*#++#%%%%#*{0}++++++++++++++++++++++
+    {0}++++++++++++++++++++++++++{2}*%%#{0}+++++++++++++++++++++++++
+    {0}++++++++++++++++++++++++++{2}+##{0}++++++++++++++++++++++++++
+    {0}+++++++++++++++++++++++++++++++++++++++++++++++++++++++ 
+  colors:
+    ansi:
+      - white
+      - black
+      - orange
+    hex:
+      - "#FFFFFF"
+      - "#000000"
+      - "##ff5a04"
```

```suggestion
      - "#FF5A04"
```
Extra `#`, and made capitalization consistent with the other colors.

---

I fix this in my last commit